### ISSUE TITLE
About result

### ISSUE THREAD

**Issue by BCXNSD** [vidit09/domaingen#34]
I use daytime_clear_train as the training set, Day-Clear_test，Night-Clear_train，Dusk-Rainy_train，Night-Rainy_train，Day-Foggy_train As a test set, but compared to the original paper, my experimental results showed significant drops in all datasets except for the Day Clear_test dataset. May I ask why this is?
![image](https://github.com/user-attachments/assets/8b791931-132c-4df5-9a40-7da263e101e5)

**Comment by vidit09** [comment]
Hello, 
Could you check if you're able to reproduce then numbers with the provided weights? To sanity check the setup.
Otherwise, we ran all of our experiments on a single GPU with the mentioned batch size(on config files), used the best model on validation set of source domain to evaluate on other domains.

**Comment by BCXNSD** [comment]
> Hello, Could you check if you're able to reproduce then numbers with the provided weights? To sanity check the setup. Otherwise, we ran all of our experiments on a single GPU with the mentioned batch size(on config files), used the best model on validation set of source domain to evaluate on other domains.

yes, I can reproduce the numbers with the provided weights. But i use the V100 for training. And can i ask you how long have you take to training the model with 100000 iteration?
